docs/hands-on-learning/fundamentals/module-2-recipe-builder.md

```diff
@@ -5,7 +5,7 @@ description: How to compose a recipe with recipe builder.
 
 # Module 2: Recipe builder
 
-Once you know what recipes are available and how to run them, you may want to customize what they do. You might want to change the order of steps, remove something you don’t need, or combine several small recipes into one. The recipe builder in Moderne lets you do this without writing any code.
+Once you know what recipes are available and how to run them, you may want to customize what they do. You might want to change the order of steps, remove something you don’t need, or combine several small recipes into one. The [recipe builder](https://docs.openrewrite.org/running-recipes/customize-recipe) in Moderne lets you do this without writing any code.
```

Comment:
Rather than linking to the openrewrite docs which then link back to the Moderne docs - I feel like it would be nicer to link to the Moderne recipe builder docs: https://docs.moderne.io/user-documentation/moderne-platform/how-to-guides/new-recipe-builder/